src/components/Interviewer/Pool/IntervieweeAvailability.tsx

```diff
@@ -0,0 +1,129 @@
+            <Button
+              variant="contained"
+              onClick={() => GoToOffsetWeek(-1)}>
+              <GrPrevious />
+            </Button>
+            <h3 style={{margin: '4px 20px 0 20px'}}>
+              {monthNames[new Date((DateTime.getOffsetDateAndHour(currentStartWeek, 0, 0))).getMonth()]}
+            </h3>
+            <Button
+              variant="contained"
+              onClick={() => GoToOffsetWeek(1)}>
+              <GrNext />
+            </Button>
+          </div>
+          <TableContainer component={Paper}>
+          <Table style={divStyle} aria-label="simple table">
+            <TableHead>
+              <TableRow>
+                <TableCell align="center">Time/Day</TableCell>
+                <TableCell align="center">Sun {new Date((DateTime.getOffsetDateAndHour(currentStartWeek, 0, 0))).getDate()}</TableCell>
+                <TableCell align="center">Mon {new Date((DateTime.getOffsetDateAndHour(currentStartWeek, 1, 0))).getDate()}</TableCell>
+                <TableCell align="center">Tue {new Date((DateTime.getOffsetDateAndHour(currentStartWeek, 2, 0))).getDate()}</TableCell>
+                <TableCell align="center">Wed {new Date((DateTime.getOffsetDateAndHour(currentStartWeek, 3, 0))).getDate()}</TableCell>
+                <TableCell align="center">Thu {new Date((DateTime.getOffsetDateAndHour(currentStartWeek, 4, 0))).getDate()}</TableCell>
+                <TableCell align="center">Fri {new Date((DateTime.getOffsetDateAndHour(currentStartWeek, 5, 0))).getDate()}</TableCell>
+                <TableCell align="center">Sat {new Date((DateTime.getOffsetDateAndHour(currentStartWeek, 6, 0))).getDate()}</TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {dayHours.filter(d => isTimeReservedInAnyDayByCandidate(DateTime.getUtcDay(d))).map((hour: any) => (
+                <TableRow key={hour}>
+                  <TableCell component="th" scope="row">
+                    {DateTime.getUtcDay(hour)}
+                  </TableCell>
+                  {days.map((day: any) => (
+                    <TableCell component="th" scope="row" key={DateTime.getOffsetDateAndHour(currentStartWeek, day, hour)}>
+                      {isTimeReservedByCandidate(DateTime.getOffsetDateAndHour(currentStartWeek, day, hour)) ? 
+                        <FormControlLabel onChange={() => {
+                          setSelectedDateOfInterview(DateTime.getOffsetDateAndHour(currentStartWeek, day, hour));
+                        }} value={DateTime.getOffsetDateAndHour(currentStartWeek, day, hour)} control={<Radio />} label='' />
+                        : null}
+                    </TableCell>
+                  ))}
+              </TableRow>
+              ))}
+            </TableBody>
+          </Table>
+        </TableContainer>
+      </RadioGroup>
+      </FormControl>
+    </UIMainContainer>
+  );
+};
+
+export default IntervieweeAvailability;
```

Comment:
So far, the split in screen and components I'm not pretty sure if is required for this. cc/ @GoberInfinity